src/main/java/com/happeningnow/happeningnow/Application.java

```diff
@@ -2,19 +2,12 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
 
-@RestController
+
 @SpringBootApplication
 public class Application {
-	@RequestMapping("/")
```

Comment:
`/` has the same effect as  empty, so, in cases like this, you can use just `@RequestMapping`

Reply:
Ok